From 270750379c6c75bf284647c5c606b43ffaceeabb Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyunk@xilinx.com>
Date: Mon, 11 Jun 2018 17:50:16 +0530
Subject: [PATCH 0361/1852] drm: xlnx: vtc: Remove xilinx_vtc_remove()

commit 3dc5aa34a342132401b8f89a25699182a19886bc from
https://github.com/Xilinx/linux-xlnx.git

Remove xilinx_vtc_remove() as devm_* managed functions are used
in xilinx_vtc.c.

Signed-off-by: Hyun Kwon <hyunk@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_vtc.c | 6 ------
 drivers/gpu/drm/xlnx/xlnx_vtc.h | 1 -
 2 files changed, 7 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_vtc.c b/drivers/gpu/drm/xlnx/xlnx_vtc.c
index c44e3af406f2..dc3ba2c7209e 100644
--- a/drivers/gpu/drm/xlnx/xlnx_vtc.c
+++ b/drivers/gpu/drm/xlnx/xlnx_vtc.c
@@ -572,9 +572,3 @@ struct xilinx_vtc *xlnx_vtc_probe(struct device *dev,
 
 	return vtc;
 }
-
-/* remove vtc */
-void xilinx_vtc_remove(struct xilinx_vtc *vtc)
-{
-	xlnx_vtc_reset(vtc);
-}
diff --git a/drivers/gpu/drm/xlnx/xlnx_vtc.h b/drivers/gpu/drm/xlnx/xlnx_vtc.h
index 6da57a3547d2..533abbf932c9 100644
--- a/drivers/gpu/drm/xlnx/xlnx_vtc.h
+++ b/drivers/gpu/drm/xlnx/xlnx_vtc.h
@@ -36,6 +36,5 @@ struct device_node;
 
 struct xilinx_vtc *xlnx_vtc_probe(struct device *dev,
 				    struct device_node *node);
-void xilinx_vtc_remove(struct xilinx_vtc *vtc);
 
 #endif /* _XILINX_VTC_H_ */
-- 
2.31.1

